groups/reconcile.go

```diff
@@ -246,40 +246,48 @@ func main() {
 	}
 }
 
-func readConfig(configFilePath string, confirmChanges bool) error {
-	if *verbose {
-		log.Printf("reading config file %s", configFilePath)
-	}
+func (c *Config) Load(configFilePath string, confirmChanges bool) error {
+	log.Printf("reading config file: %s", configFilePath)
```

Comment:
nit: these logs are enabled at other places only if `verbose` is true. Is there a reason why we remove that check here?

Reply:
I don't know that I accomplished this cleanly, but what I tried to do was have top-level config type stuff output by default, and leave verbose for individual operations.  The reason being that I, personally, find a lot more comfort in getting a sense of what this is going to do and where it's reading from before it does things... but then I only want to know when it's making changes (vs. walking though things)

I realize in running the tests this makes things a little noisy for non-CLI-invocation use-cases.  But the tests are it, for now, and there aren't many.

Were someone to keep going on refactoring this for testability, I would suggest using an actual logging package with verbosity levels, such that the really granular stuff could be dropped to a higher verbosity level (debug or v=2 or something), and configuration of that logging to be done by the main entrypoint vs. a much quieter default used by tests.